front/app/containers/App/constants.ts

```diff
@@ -137,6 +140,7 @@ export const shortenedAppLocalePairs = {
   'it-IT': 'Italiano',
   'kl-GL': 'Kalaallisut',
   'lb-LU': 'Lëtzebuergesch',
+  'lv-LV': 'Latviski',
```

Comment:
I'm just curious, where have you found this name?

Reply:
Google translate.

Reply:
Hm that's complicated and confusing, but it seems it should be "latviešu" https://en.wikipedia.org/wiki/List_of_language_names

In my personal experience, Google translate is good for texts, but not for specific words. And of course, I can be wrong in this case :smile: I decided to double-check without any real reason, just because "Latviski" looks like not Latvian but a Slavic word.

But anyway we can change it later if they complain.

Reply:
I can change it here. Thanks for the info!